keras_cv/layers/preprocessing/rand_augment.py

```diff
@@ -0,0 +1,295 @@
+            )
+        self.magnitude_stddev = float(magnitude_stddev)
+
+    def _augment(self, sample):
+        sample["images"] = preprocessing_utils.transform_value_range(
+            sample["images"], self.value_range, (0, 255)
+        )
+        result = super()._augment(sample)
+        result["images"] = preprocessing_utils.transform_value_range(
+            result["images"], (0, 255), self.value_range
+        )
+        return result
+
+    @staticmethod
+    def get_standard_policy(value_range, magnitude, magnitude_stddev, seed=None):
+        policy = create_rand_augment_policy(magnitude, magnitude_stddev)
+
+        auto_contrast = cv_preprocessing.AutoContrast(
+            **policy["auto_contrast"], value_range=value_range, seed=seed
+        )
+        equalize = cv_preprocessing.Equalization(
+            **policy["equalize"], value_range=value_range, seed=seed
+        )
+
+        solarize = cv_preprocessing.Solarization(
+            **policy["solarize"], value_range=value_range, seed=seed
+        )
+        solarize_add = cv_preprocessing.Solarization(
+            **policy["solarize_add"], value_range=value_range, seed=seed
+        )
+        invert = cv_preprocessing.Solarization(
+            **policy["invert"], value_range=value_range, seed=seed
+        )
+
+        color = cv_preprocessing.RandomColorDegeneration(**policy["color"], seed=seed)
+        contrast = cv_preprocessing.RandomContrast(**policy["contrast"], seed=seed)
+        brightness = cv_preprocessing.RandomBrightness(
+            **policy["brightness"], seed=seed
+        )
+        shear_x = cv_preprocessing.RandomShear(**policy["shear_x"], seed=seed)
+        shear_y = cv_preprocessing.RandomShear(**policy["shear_y"], seed=seed)
+        translate_x = cv_preprocessing.RandomTranslation(
+            **policy["translate_x"], seed=seed
+        )
+        translate_y = cv_preprocessing.RandomTranslation(
+            **policy["translate_y"], seed=seed
+        )
+        cutout = cv_preprocessing.RandomCutout(**policy["cutout"], seed=seed)
+        return [
+            auto_contrast,
+            equalize,
+            solarize,
+            solarize_add,
+            invert,
+            color,
+            contrast,
+            brightness,
+            shear_x,
+            shear_y,
+            translate_x,
+            translate_y,
+            cutout,
+        ]
+
+    def get_config(self):
+        config = super().get_config()
+        config.update(
+            {
+                "value_range": self.value_range,
+                "augmentations_per_image": self.augmentations_per_image,
+                "magnitude": self.magnitude,
+                "magnitude_stddev": self.magnitude_stddev,
+                "rate": self.rate,
+                "seed": self.seed,
+            }
+        )
+        # layers is recreated in the constructor
+        del config["layers"]
```

Comment:
Are the layers created from parent class?

Reply:
Yeah, if we don't del this RandAugment serialization fails as it tries to provide `layers` twice in the constructor.